domains/asuid.dankamigos.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "mokshdk",
+        "email": "anirudhsus001@gmail.com"
+    },
+    "record": {
+        "TXT": [
+            "562965D131E7C913D1796476301A86A148A2B64B52D99FC316724B0B785FAAB1"
+        ]
```

Comment:
TXT records alone don't do anything for standalone domains. Unless they specifically need this for verification, you're likely doing your record wrong.